aws_lambda_powertools/utilities/parameters/base.py

```diff
@@ -230,6 +231,8 @@ def transform_value(value: str, transform: str, raise_on_transform_error: bool =
             return json.loads(value)
         elif transform == TRANSFORM_METHOD_BINARY:
             return base64.b64decode(value)
+        elif transform == TRANSFORM_METHOD_YAML:
+            raise NotImplementedError
```

Comment:
We're adding the `TRANSFORM_METHOD_YAML` on line 20, but the implementation is not defined. Should we remove both and put it in a different PR at a later stage?